run_md.py

```diff
@@ -0,0 +1,879 @@
+                             'calculations will run on a single machine as usual.')
+    parser.add_argument('-c', '--ncpu', metavar='INTEGER', required=False, default=cpu_count(), type=int,
+                        help='number of CPU per server. Use all cpus by default.')
+    parser.add_argument('--topol', metavar='topol.top', required=False, default=None, type=filepath_type,
+                        help='Required if gro file of the protein is provided')
+    parser.add_argument('--topol_itp', metavar='topol_chainA.itp topol_chainB.itp', required=False, nargs='+', default=None, type=filepath_type,
+                        help='if protein has multiple chain there are multiple topologies for each of them. Itp files for each chain are required')
+    parser.add_argument('--posre', metavar='posre.itp', required=False, nargs='+', default=None, type=filepath_type,
+                        help='required if gro file of the protein is provided. Can be multiple files for each chain')
+    parser.add_argument('--md_time', metavar='ns', required=False, default=1, type=float,
+                        help='time of MD simulation in ns')
+    parser.add_argument('--npt_time', metavar='ps', required=False, default=100, type=int,
+                        help='time of NPT equilibration in ps')
+    parser.add_argument('--nvt_time', metavar='ps', required=False, default=100, type=int,
+                        help='time of NVT equilibration in ps')
+    parser.add_argument('--not_clean_log_files',  action='store_true', default=False,
+                        help='Not to remove all backups of md files')
+    #continue md
+    parser.add_argument('--tpr', metavar='FILENAME', required=False, default=None, type=filepath_type,
+                        help='tpr file from the previous MD simulation')
+    parser.add_argument('--cpt', metavar='FILENAME', required=False, default=None, type=filepath_type,
+                        help='cpt file from previous simulation')
+    parser.add_argument('--xtc', metavar='FILENAME', required=False, default=None, type=filepath_type,
+                        help='xtc file from previous simulation')
+    parser.add_argument('--wdir_to_continue', metavar='DIRNAME', required=False, default=None, nargs='+', type=filepath_type,
+                        help='''directories for the previous simulations. Use to extend or continue the simulation. '
+                             Should consist of: tpr, cpt, xtc files''')
+    parser.add_argument('--deffnm', metavar='preffix for md files', required=False, default='md_out',
+                        help='''preffix for the previous md files. Use to extend or continue the simulation.
+                        Only if wdir_to_continue is used. Use if each --tpr, --cpt, --xtc arguments are not set up. 
+                        Files deffnm.tpr, deffnm.cpt, deffnm.xtc will be used from wdir_to_continue''')
```

Comment:
it seems that MD can be continued only for a single case. Do we need MD continuation for multiple cases at the same time?